client/views/msg.tpl

```diff
@@ -1,6 +1,6 @@
 {{#each messages}}
 <div class="msg {{type}} {{#if self}}self{{/if}}">
-	<span class="time">
+	<span class="time" data-time="{{time}}">
```

Comment:
Easy place to keep the full timestamp so we can re-render later on if they change the format.